### PR TITLE
d3 ciel 3_1, 3_2*

### PR DESCRIPTION
**Pull request by Avuxo**

### CODE REVIEW

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt [hunk 1]
```diff
@@ -271,7 +272,7 @@ I sit down on the tatami mat and leisurely enjoy a cup of tea with her.
 [sha:b094d7362abcf46e5d30554d0a61d34968e7b6c4]{
 -- Page 77, Offset 5035.
 -- 遠野くん、新しいお家の方はどうですか？　なにか大変な事になっていたみたいですけど」
-Tohno-kun, how is your new home? It seems that something serious has happened..."
+Tohno-kun, what's going on with your new home? It seemed like something serious had happened..."
```
Comment (nolo16) [inline script/Ciel/Day 3/03_01_CIEL03_2_3.txt:275]: i think you can drop "had" here

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt [hunk 2]
```diff
@@ -352,7 +353,7 @@ There have been eight victims already, so the police will probably catch the cul
 -- Page 85, Offset 5051. Mods: @n.
 -- 「もうっ。そんなコトを言っていると寝首をかかれますよ。
 // Lit. expression 'to cut the head off a sleeping person'
-"Oh, no. If you keep talking like that, you'll get attacked while your guard is down.
+"Come on. If you keep talking like that, you'll get attacked while your guard is down.
```
Comment (nolo16) [inline script/Ciel/Day 3/03_01_CIEL03_2_3.txt:356]: be attacked

File: script/Ciel/Day 3/03_01_CIEL03_1.txt [hunk 3]
```diff
@@ -11,7 +11,7 @@ I wasn't used to the route, so I must have paced myself wrong.
 [sha:6d00dc71bcca4185f381b831e68bdb3c4e384a44]{
 -- Page 1, Offset 4854.
 -- 朝の雑談時間も終わりにさしかかり、生徒の大部分は自分の机に座っている。
-The morning chatter is starting to dwindle, and most of the students have gone back to their desks.
+The morning chatter is starting to dwindle, and most of the students are returning to their desks.
```
Comment (nolo16) [inline script/Ciel/Day 3/03_01_CIEL03_1.txt:14]: prefer original here imo

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 4]
```diff
@@ -401,29 +404,28 @@ As I watch the two of them out of the corner of my eye, I gaze idly at the scene
...
 -- Page 44, Offset 4960. Mods: @k, @e.
 -- 「そんなことはないッス。先輩は他のおんなどもに比べて発育いいんスから。
-// 摂理 is the actual word but I guess fuel efficient makes more sense
 "That's not true. You're a lot more developed than the other girls.
 }
 [sha:5640c1d8727735d842bd3d7891e9dd0a7867b42d]{
 -- Page 44, Offset 4962. Glued. Mods: @x.
 -- ……が大きいぶん、燃費が悪いのは自然の摂理っスよ」
-// 摂理 is the actual word but I guess fuel efficient makes more sense
+// I have no idea how to work in 摂理
```
Comment (nolo16) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:412]: It's God's will that the bigger you are, the less fuel-efficient you are, ya know? (kidding)

no idea either, the current one seems fine imo

**Review by nolo16 — APPROVED** [review]

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt (hunk 2 above)
Comment (Avuxo) [reply script/Ciel/Day 3/03_01_CIEL03_2_3.txt:356]: can't say I agree. "get attacked" sounds more natural to me

File: script/Ciel/Day 3/03_01_CIEL03_1.txt (hunk 3 above)
Comment (Avuxo) [reply script/Ciel/Day 3/03_01_CIEL03_1.txt:14]: Gotta disagree. "return to your seats" is a set phrase in a school environment.

**Pull request merged**